charts/victoria-logs-single/values.yaml

```diff
@@ -359,3 +359,12 @@ fluent-bit:
 
 # -- Add extra specs dynamically to this chart
 extraObjects: []
+
+dashboards:
+  # -- Create VictoriaLogs dashboards
```

Comment:
Maybe we should also support creating it as a GrafanaDashboard CR like k8s-stack [does](https://github.com/VictoriaMetrics/helm-charts/blob/ac89332d907dd7da7843541c52018bb36e06394c/charts/victoria-metrics-k8s-stack/templates/grafana/dashboard.yaml#L41), wdyt?

And I think we should disable it by default now, currently it's not clear or convenient to tell users how to get access to dashboard or rule in vlogs chart, since the chart doesn't contain grafana or ruler components.

Reply:
added this as well